web/modules/custom/dpl_loans/src/Plugin/Block/LoanListBlock.php

```diff
@@ -146,8 +146,8 @@ public function build() {
       'loan-list-status-badge-danger-text' => $this->t("Expired", [], ['context' => 'Loan list']),
       'loan-list-status-badge-warning-text' => $this->t("Expiring soon", [], ['context' => 'Loan list']),
       'loan-list-title-text' => $this->t("Your loans", [], ['context' => 'Loan list']),
-      'loan-list-to-be-delivered-digital-material-text' => $this->t("Due date @date", [], ['context' => 'Loan list']),
-      'loan-list-to-be-delivered-text' => $this->t("Due date @date", [], ['context' => 'Loan list']),
+      'loan-list-to-be-delivered-digital-material-text' => $this->t("Due date digital @date", [], ['context' => 'Loan list']),
+      'loan-list-to-be-delivered-text' => $this->t("Due date physical @date", [], ['context' => 'Loan list']),
```

Comment:
Generally I am not too keen on having texts which are not in a form which is suitable for end users as the source. 

In these cases it would be preferable to either see if we can find a more suitable text or use the context to differ.  I do not have a good solution in the current case so let's just go with this for now.